src/providers/bitcoin/BitcoinLedgerProvider.js

```diff
@@ -110,14 +110,17 @@ export default class BitcoinLedgerProvider extends LedgerProvider {
     return ((numInputs * 148) + (numOutputs * 34) + 10) * feePerByte
   }
 
-  async getUtxosForAmount (amount, feePerByte = 3) {
+  async getUtxosForAmount (amount) {
     const utxosToUse = []
     let addressIndex = 0
     let currentAmount = 0
     let numOutputsOffset = 0
 
     while ((currentAmount < amount)) {
-      const address = await this.getAddressFromIndex(addressIndex)
+      const [ feePerByte, address ] = await Promise.all([
+        this.getMethod('getFeePerByte')(2),
```

Comment:
Why 2 blocks and not 1?

Reply:
It was just a default value i went with. Best was would be accepting user's preferred feerate/number of block from `sendTransaction` and `getUtxosForAmount`. We can look into that.